common/functions/src/scalars/arithmetics/result_type.rs

```diff
@@ -0,0 +1,724 @@
+// Code generated by common/codegen/src/main.go. DO NOT EDIT.
+//
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_datavalues::DFPrimitiveType;
```

Comment:
Add Comments:

1. Don't edit, Readonly notes.

2. How it comes from.

Reply:
> Add Comments:
> 
> 1. Don't edit, Readonly notes.
> 2. How it comes from.

done